apps/web/src/app/(pages)/(dashboard)/companies/company/page.tsx

```diff
@@ -0,0 +1,99 @@
+"use client";
+
+import { useEffect } from "react";
+import Image from "next/image";
+import { useSearchParams } from "next/navigation";
+
+import { cn } from "@cooper/ui";
+
+import { CompanyAbout } from "~/app/_components/companies/company-about";
+import { CompanyReview } from "~/app/_components/companies/company-reviews";
+import LoadingResults from "~/app/_components/loading-results";
+import NoResults from "~/app/_components/no-results";
+import { RoleCardPreview } from "~/app/_components/reviews/role-card-preview";
+import { api } from "~/trpc/react";
+
+export default function Company() {
+  const searchParams = useSearchParams();
+  const companyID = searchParams.get("id"); // Get 'id' from the URL
+
+  const company = api.company.getById.useQuery({ id: companyID ?? "" });
+  const roles = api.role.getByCompany.useQuery({ companyId: companyID ?? "" });
+
+  useEffect(() => {
+    console.log("Company ID:", companyID); // Debugging: Check if id is retrieved correctly
+  }, [companyID]);
+
+  return (
+    <>
+      {company.isSuccess ? (
+        <div className="mx-auto h-[86dvh] w-full max-w-[66dvw] gap-4 font-sans">
+          <div className="mb-6 mt-6 flex items-center justify-between">
+            <div className="flex items-center">
+              <div className="mr-3 flex h-16 w-16 items-center justify-center">
+                <Image
+                  src={`https://logo.clearbit.com/${company.data?.name.replace(/\s/g, "")}.com`}
+                  width={80}
+                  height={80}
+                  alt={`Logo of ${company.data?.name}`}
+                  className="rounded-md"
+                />
+              </div>
+              <div>
+                <h1 className="text-4xl font-bold">{company.data?.name}</h1>
+                <p className="text-lg text-gray-600">
+                  Co-op · {company.data?.industry.toLowerCase()}
```

Comment:
Check the file :
packages/db/src/schema/misc.ts
To see what industry can be. Then in the frontend utils (stringHelpers.ts) make a function that takes in the db representation and makes it pretty ◡̈ (note e.g. FOODANDBEVERAGE should be Food and Beverage)